src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -986,7 +986,7 @@ internal override IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperat
             // The CSharp operation factory assumes that UnboundLambda will be bound for error recovery and never be passed to the factory
             // as the start of a tree to get operations for. This is guaranteed by the builder that populates the node map, as it will call
             // UnboundLambda.BindForErrorRecovery() when it encounters an UnboundLambda node.
-            Debug.Assert(result.Kind != BoundKind.UnboundLambda);
+            Debug.Assert(result?.Kind != BoundKind.UnboundLambda);
```

Comment:
made this change just to make test pass. but by looking at the code, result can be null, so adding ?. seems right change.